packages/discord.js/src/structures/GuildMember.js

```diff
@@ -63,7 +64,7 @@ class GuildMember extends Base {
        * The user that this guild member instance represents
        * @type {?User}
        */
-      this.user = this.client.users._add(data.user, true);
+      this.user = this.user ? this.user._clone().patch(data.user) : new User(this.client, data.user);
```

Comment:
What's the justification for the clone here?

Reply:
Was to ensure that cloned member instances had unique user instances, gonna handle this better in _clone now though